tests/e2e/e2e_pod_client_server_test.go

```diff
@@ -21,13 +21,17 @@ var _ = OSMDescribe("Test HTTP traffic from 1 pod client -> 1 pod server",
 		Bucket: 1,
 	},
 	func() {
-		Context("SimpleClientServer with a Kubernetes Service for the Source", func() {
+		Context("SimpleClientServer with a Kubernetes Service for the Source: HTTP", func() {
 			testTraffic(true)
 		})
 
-		Context("SimpleClientServer without a Kubernetes Service for the Source", func() {
+		Context("SimpleClientServer without a Kubernetes Service for the Source: HTTP", func() {
 			testTraffic(false)
 		})
+
+		Context("SimpleClientServer without a Kubernetes Service for the Source: TCP", func() {
```

Comment:
`testTCPTraffic()`, though similar setup, doesn't really share any code with `testTraffic`, since we are voucing to keep a test-per-file too, I'd suggest to move it on a different file altogether and have the `*_http_test.go` and the `*_tcp_test.go` versions of the test on separate files

Reply:
The reason I left it here is because `testTCPTraffic` falls under the bucket of `e2e_pod_client_server_test` where simple client server traffic scenario is tested. We can break this into `e2e_pod_client_server_test_<http|tcp>` if necessary, but I don't see a strong reason to do so at this moment. Should be a straightforward change though (mostly moving files around). I'd rather address this in a follow-up if it must be done.